src/clr-addons/location-bar/location-bar-search/location-bar-search.component.ts

```diff
@@ -77,4 +73,8 @@ export class LocationBarSearchComponent implements OnInit {
   preventDropdownActions(event: Event) {
     event.stopPropagation();
   }
+
+  displayMinCharactersText(): string {
```

Comment:
```suggestion
  getMinCharactersText(): string {
```

the method doesn't display anything, but returns it :) therefore "get" prefix fits better